components/table/stories/table.test.js

```diff
@@ -1,162 +1,174 @@
 import { Variants } from "@spectrum-css/preview/decorators";
 import { Template } from "./template.js";
 
+const ExampleSummarySelectedContent = [
+	{
+		cellContent: "Table Row Alpha",
```

Comment:
```suggestion
		cellContent: "Table row alpha",
```

Should we take this opportunity to update the title case here to match the style guide?